### ISSUE TITLE
Make links in outputs clickable

### ISSUE THREAD

**Issue by tracyspacy** [Electron-Cash/Electron-Cash#1037]
As Electron-Cash shows OP_RETURN memos in outputs as default, I propose to make hyperlinks there clickable.
![electron-cash_op_return message with hyperlink](https://user-images.githubusercontent.com/42025315/49896675-63362180-fe54-11e8-87bd-2ebcce61123d.png)

We in memopay.xyz pushing microtransactions along with ad message in op_return to BCH addresses, normally such ad messages contain links, blockexplorers already made them clickable, and it would be great to have clickable links in EC wallet too.

I suppose clickable links will be useful for other services using op_return and regular users too (as EC wallet already allows users to add op_return memos to transactions).

**Comment by markblundeberg** [comment]
>blockexplorers already made them clickable

Which block explorers do this?

**Comment by tracyspacy** [comment]
> > blockexplorers already made them clickable
> 
> Which block explorers do this?
[blockchair](https://blockchair.com/bitcoin-cash/transaction/29d1a7893dca78465162e4fd5b2bb2eab3f4909f4b83cbe2f0a3c6a7f5461368) and [explorer.bitcoin.com](https://explorer.bitcoin.com/bch/tx/e2463f0add330a65abfe03e93b09a2c6b01b979763b1c248f4f28e6b47994ae2 )

**Comment by cculianu** [comment]
@tracyspacy  @markblundeberg  

So these guys want us to do free work for them so they can spam the blockchain with ads. 😛 

Throw us a little BCH love if you want us to help you put ads on the blockchain and we can get it done in under a week.

Otherwise -- we are mostly volunteers here -- we may not get around to it for a while.

^^ Donation address is above in this thread.

Thanks!

**Comment by cculianu** [comment]
This is a good suggestion .. Bumping for visibility.

**Comment by markblundeberg** [comment]
I agree in principle. But if this is done, let's make very sure it's not vulnerable to XSS (html injection) attacks. So, make good use of https://docs.python.org/3/library/html.html#html.escape . :-)

**Comment by cculianu** [comment]
Definitely good point.
